pwnlib/exception.py

```diff
@@ -13,13 +13,14 @@ def __init__(self, msg, reason = None, exit_code = None):
         Exception.__init__(self, msg)
         self.reason = reason
         self.exit_code = exit_code
+        self.message = msg
 
     def __repr__(self):
         s = 'PwnlibException: %s' % self.message
         if self.reason:
             s += '\nReason:\n'
             s += ''.join(traceback.format_exception(*self.reason))
-        elif sys.exc_type not in [None, KeyboardInterrupt]:
+        elif sys.exc_info() not in [None, KeyboardInterrupt]:
```

Comment:
This will never be true; exc_info() returns a 3-tuple
```suggestion
        elif sys.exc_info()[0] not in [None, KeyboardInterrupt]:
```